lib/Sema/TypeCheckProtocol.h

```diff
@@ -347,6 +347,20 @@ class OptionalAdjustment {
   SourceLoc getOptionalityLoc(TypeRepr *tyR) const;
 };
 
+/// Describes the position for optional adjustment made to a witness
```

Comment:
Can you mention the diagnostic that this enum matches up with?

Reply:
Also since it's not used anywhere else, you might as well put this enum in an anonymous namespace above the function that returns it.

Reply:
Sure, I have done that now.